tests/Struct/FindologicProductTest.php

```diff
@@ -1044,6 +1044,46 @@ function (SalesChannelDomainEntity $domain) use ($salesChannel) {
         $this->assertEquals($expectedUrl, $findologicProduct->getUrl());
     }
 
+    public function testEmptyCategoryNameShouldStillExportCategory()
+    {
+        $mainCatId = $this->getContainer()->get('category.repository')
+            ->searchIds(new Criteria(), Context::createDefaultContext())->firstId();
+
+        $categoryId = Uuid::randomHex();
+        $productEntity = $this->createTestProduct([
+            'categories' => [
+                [
+                    'parentId' => $mainCatId,
+                    'id' => $categoryId,
+                    'name' => ' ',
```

Comment:
Is empty string also possible to configure or even null?

Reply:
Blank or null values will be prevented.

![image](https://user-images.githubusercontent.com/20555112/98669487-ba4f1480-2351-11eb-85f7-7adc4a848853.png)

Reply:
This means the category of the affected client is configured with one space or similar?

Reply:
The category of the affected client has no translation. Setting up translations is not an easy task when dealing with them in tests, so I just assume that the configured "default language" has an empty string assigned, which results in the same.

Reply:
Alright